Add test with multiple ops

priyaramani opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #73888

Differential Revision: [D34702376](https://our.internmc.facebook.com/intern/diff/D34702376)

priyaramani committed:
Add test with multiple ops

[ghstack-poisoned]

priyaramani commented:
@priyaramani has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34702376).

larryliu0820 commented on test/mobile/lightweight_dispatch/test_codegen_unboxing.cpp:
```diff
@@ -190,6 +191,30 @@ TEST(LiteInterpreterTest, DivideTensor) {
 }
+
+TEST(LiteInterpreterTest, MultipleOps) {
+  // Load check in model: basic.ptl
```
Maybe change this?

larryliu0820 commented on test/mobile/lightweight_dispatch/test_codegen_unboxing.cpp:
```diff
@@ -190,6 +191,30 @@ TEST(LiteInterpreterTest, DivideTensor) {
+  //               super(Model, self).__init__()
+  //               self.ops = torch.nn.Sequential(
+  //                   torch.nn.ReLU(),
+  //                   torch.nn.AdaptiveAvgPool2d(output_size=(1, 1)),
```
Also we don't have this op yet, right?

larryliu0820 reviewed (approved):
Looks good to me! Please address the comments before land.

priyaramani committed:
Update on "Add test with multiple ops"


Differential Revision: [D34702376](https://our.internmc.facebook.com/intern/diff/D34702376)

[ghstack-poisoned]
